engine/src/main/java/org/camunda/bpm/engine/impl/db/EnginePersistenceLogger.java

```diff
@@ -837,11 +838,16 @@ public void errorFetchingTelemetryInitialMessagePropertyInDatabase(Exception exc
         "Error while fetching the telemetry initial message status property from the database: {}", exception.getMessage());
   }
 
-	
+
   public void logTaskWithoutExecution(String taskId) {
     logDebug("108",
       "Execution of external task {} is null. This indicates that the task was concurrently completed or deleted. "
       + "It is not returned by the current fetch and lock command.",
       taskId);
   }
+
+  public void errorEventDefinitionEvaluationException(String taskId, CamundaErrorEventDefinition errorEventDefinition, Exception exception) {
+    logDebug("109", "Evaluation of error event definition's expression {} on external task {} failed and will be considered as not-fulfilled. "
```

Comment:
Thanks for that hint. 😃 
I had the same impulse, discussed this with Thorben and we came to the conclusion that this is documented and expected engine behavior, so logging this as a warning/error would be a bit weird.
Let me know if you still think otherwise, I am completely open for a debate on that and have no hard feeling here which level to choose. 😃

Reply:
Thanks for the call. 🙂 👍  Now I understand why logging it on a more severe level is not ideal regarding the RPA bridge use case. As discussed, let's add the log message to the `ExternalTaskLogger`. Like this, it might be easier for the user to find out what's wrong while developing when not using the RPA bridge.